index.js

```diff
@@ -690,6 +687,10 @@ class UserAgent {
         headers['OAuth-Token'] = response.body.access_token;
         this._setState('headers', headers);
         this._setState('refreshToken', response.body.refresh_token);
+
+        if (this.username === Agent.ADMIN) {
+            Fixtures._storeAuth(response, false);
```

Comment:
The `false` here is important. If it was left to the default, we would have infinite mutual recursion as both `Fixtures._storeAuth` and `UserAgent._updateAuthState` would end up calling each other over and over again.

However, we do want the default value to be `true`, because that ensures that all logins and refreshes do call both methods.